src/openstack_cloud/openstack_manager.py

```diff
@@ -1178,8 +1206,18 @@ def reconcile(self, quantity: int) -> int:
                     ) from exc
 
                 logger.info("Creating %s OpenStack runners", delta)
-                for _ in range(delta):
-                    self._create_runner(conn)
+                args = [
+                    OpenstackRunnerManager._CreateRunnerArgs(
+                        self._cloud_config, self.app_name, self.unit_num, self._config
+                    )
+                    for _ in range(delta)
+                ]
+                with Pool(processes=min(delta, 10)) as pool:
```

Comment:
Out of curiosity, what happens if there are more runners that are required than the number of processes spawned by the pool?